contracts/fio.address/fio.address.cpp

```diff
@@ -1735,8 +1732,45 @@ namespace fioio {
             }
             else check(false, "Failed to decrement eligible bundle counter"); // required to fail the parent transaction
         }
+
```

Comment:
This should not be a standard action. Needs to be inline for ease of use within other contracts.

Reply:
I am not sure how that is done. Could you point me to an example of how to define it as an inline and how it is called from another contract?

Reply:
Since the require_auth is set on this, shouldn't be an issue.